### PR TITLE
Dashboard: Support for drag api for add card menu

### PR DESCRIPTION
**Pull request by samiyaakhtar**
#### Pull request checklist

- [x] Include a change request file using `$ npm run change`

#### Description of changes

We need the ability to support adding cards outside the dashboard grid layout by dragging into the grid, for the right side add menu which will let us add cards into our dashboard. This is not supported out of the box by react-grid-layout, so we've created a fork of the library which we are referencing in this PR. After a discussion with fabric team we decided that for now while this is an experiment, we will reference the fork we created and in future move it to somewhere more permanent. This is critical for us to get moving with our add card feature in dashboard.

### CODE REVIEW

File: packages/dashboard/package.json [hunk 1]
```diff
@@ -5,15 +5,14 @@
...
-    "@types/react-grid-layout": "^0.16.4",
     "@uifabric/charting": "^0.4.0",
     "@uifabric/example-app-base": ">=6.5.1 <7.0.0",
     "@uifabric/experiments": ">=6.17.0 <7.0.0",
     "css-loader": "^0.15.1",
     "office-ui-fabric-react": ">=6.37.1 <7.0.0",
     "react": "^16.4.1",
     "react-dom": "^16.4.1",
-    "react-grid-layout": "^0.16.6",
+    "react-grid-layout": "samiyaakhtar/react-grid-layout",
```
Comment (venkatna) [inline packages/dashboard/package.json]: > "react-grid-layout": "samiyaakhtar/react-grid-layout", [](start = 3, length = 55)

Don't need types anymore?

File: apps/fabric-website/src/pages/Styles/LayoutPage/LayoutPage.tsx [hunk 2]
```diff
@@ -146,7 +146,8 @@ export class LayoutPage extends React.Component<any, any> {
             devices. The columns in a row should add up to 12 for each device size.
           </p>
           <p>
-              Newer versions of Fabric require the <code>dir</code> attribute to be set to specify how the content should be rendered (whether left-to-right, <code>ltr</code>, or right-to-left, <code>rtl</code>).
+            Newer versions of Fabric require the <code>dir</code> attribute to be set to specify how the content should
+            be rendered (whether left-to-right, <code>ltr</code>, or right-to-left, <code>rtl</code>).
```
[the remark this answers is not in the document]
Comment (samiyaakhtar) [reply apps/fabric-website/src/pages/Styles/LayoutPage/LayoutPage.tsx]: My text editor seems to make a bunch of indenting changes to these files, will revert.

File: apps/fabric-website/webpack.config.js [hunk 3]
```diff
@@ -36,11 +36,11 @@ module.exports = function (argv) {
 
       externals: [
         {
-          'react': 'React'
+          react: 'React'
```
[the remark this answers is not in the document]
Comment (samiyaakhtar) [reply apps/fabric-website/webpack.config.js]: This came as part of a merge as well, not sure why it shows under my PR. I did merge

File: packages/dashboard/package.json (hunk 1 above)
Comment (samiyaakhtar) [reply packages/dashboard/package.json]: Yes. I include typings file in the forked project since we made changes to it

**Comment by samiyaakhtar** [comment]
Will make a new one.

**Pull request closed without merging**